content/templates.xqm

```diff
@@ -9,8 +10,7 @@ xquery version "3.1";
  : @contributor Joe Wicentowski
 :)
 module namespace templates="http://exist-db.org/xquery/html-templating";
-
-import module namespace inspect="http://exist-db.org/xquery/inspection";
+import module namespace inspect="http://exist-db.org/xquery/inspection" at "java:org.exist.xquery.functions.inspect.InspectionModule";
```

Comment:
The import did work without `at` before. Was it necessary to add it?

Reply:
Fixed. Removed from commit.